.github/workflows/pr.yaml

```diff
@@ -107,7 +107,7 @@ jobs:
         shell: bash
         run: tar -xzf maven-repo.tgz -C ~
       - id: files
-        uses: jitterbit/get-changed-files@v1
+        uses: tj-actions/changed-files@v34.4.3
```

Comment:
Could you stick with Major?

Could you describe why is this change needed?

Reply:
There hasn't been any [activity](https://github.com/jitterbit/get-changed-files) for this GHA `jitterbit/get-change-files` for more than 1 year and a half, and it's about to get deprecated due to Node12 not being updated. So I decided to get a more updated and active GHA replacement.